index.bs

```diff
@@ -357,59 +356,102 @@ The client facilitates these security measures by providing correct origins and
 operation. Since this is an integral part of the WebAuthn security model, user agents MUST only expose this API to callers in
 [=secure contexts=].
 
-The Web Authentication API is defined by the union of the Web IDL fragments presented in the following sections. A combined IDL listing is given in the [[#idl-index]]. The API is defined as a part of the {{Navigator}} interface:
-
-
-<pre class="idl">
-    partial interface Navigator {
-        readonly attribute WebAuthentication authentication;
-    };
-</pre>
+The Web Authentication API is defined by the union of the Web IDL fragments presented in the following sections. A combined IDL
+listing is given in the [[#idl-index]].
 
+## <dfn interface>ScopedCredential</dfn> Interface ## {#iface-scopedcredential}
 
-## <dfn interface>WebAuthentication</dfn> Interface ## {#iface-credential}
+The {{ScopedCredential}} interface inherits from {{Credential}} [[!CREDENTIAL-MANAGEMENT-1]], and contains the attributes that
+are returned to the caller when a new credential is created, or a new assertion is requested.
 
 <xmp class="idl">
     [SecureContext]
-    interface WebAuthentication {
-        Promise<ScopedCredentialInfo> makeCredential(
+    interface ScopedCredential : Credential {
```

Comment:
`Transferable` basically means that when the object is `postMessage`d, it doesn't copy the data, but moves it out of your window. We don't need that behavior for any of the current credential types, and if y'all don't need it either, then I'm happy to drop `Transferable` from the CM spec.

That said, there's nothing stopping a malicious developer from copying or serializing the `ScopedCredential` object and using `postMessage()` to transfer that data to a different window. What's the threat you'd like to defend against?